WordPress/gutenberg
# Add Dark Mode-specific help section images

## Related PRs

* https://github.com/wordpress-mobile/gutenberg-mobile/pull/3884

## Description
Add Dark Mode-specific images to the Help section to reduce harshness of visual illustrations. 

## How has this been tested?
1. Launch the editor. 
2. Open the Help section:
    - iOS: Tap the three-dot menu in the top-right corner and select Help. 
    - Android: Shake the device and select Help. 
3. Navigate to each help topic and verify the appropriate image displays for the content and Dark Mode status. 
4. Enable Dark Mode and repeat steps 1-3. 

## Screenshots 
| Light Mode | Dark Mode |
| - | - |
| ![light-ios](https://user-images.githubusercontent.com/438664/131176855-f8864c72-8535-42e3-b2fc-f091ef4e9927.PNG) | ![dark-ios](https://user-images.githubusercontent.com/438664/131176873-70e43e66-d71d-4725-a710-6d0f1271f741.PNG) |
| ![light-android](https://user-images.githubusercontent.com/438664/131176882-d6c6c356-b5c1-4d44-acd8-fd9e86b53b16.jpg) | ![dark-android](https://user-images.githubusercontent.com/438664/131176895-cd021bac-a24e-4009-804f-55a30feef2e6.jpg) |

## Types of changes
New feature

## Checklist:
